app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavScreen.kt

```diff
@@ -156,6 +158,11 @@ fun RootNavScreen(
                 )
             }
 
+            RootNavState.ExpiredRegistrationLink -> {
```

Comment:
What is the intended way to prompt this screen?

Reply:
follow up PR, but quick hits version, we get CompleteReg intent, we check the token via endpoint if valid we do what we do currently and set special circumstance to complete registration, else if expired set it to one that will cause root nav to here. I can post that PR, but was just waiting for this to get merged assuming I'd have some sub commits and saving rebase hell :)

Reply:
Gotcha, makes sense to me!